src/_nebari/upgrade.py

```diff
@@ -666,6 +666,23 @@ def _version_specific_upgrade(
         return config
 
 
+class Upgrade_2023_12_1(UpgradeStep):
+    version = "2023.12.1"
+
+    def _version_specific_upgrade(
+        self, config, start_version, config_filename: Path, *args, **kwargs
+    ):
+        rich.print("\n ⚠️  Warning ⚠️")
+        rich.print(
+            "-> Please run the [green]rm -rf stages[/green] so that we can regenerate an updated set of Terraform scripts for your deployment."
+        )
+        rich.print("\n ⚠️  Deprecation Warning ⚠️")
+        f"-> [green]{self.version}[/green] is the last Nebari version that supports the jupyterlab-videochat extension"
+        rich.print()
```

Comment:
@dcmcand You probably want to print line 680 here. Also, maybe let's add a dot at the end of the sentence to keep consistency.